go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/grafana/carbon-relay-ng
 
-go 1.20
+go 1.21.8
```

Comment:
should we just say go 1.21 and that will pick up whatever the latest go 1.21 is?